plugins/chat/app/services/chat/lookup_channel_threads.rb

```diff
@@ -93,28 +92,26 @@ def fetch_threads(guardian:, channel:, **)
             user: :user_status,
           ],
         )
-        .joins(:user_chat_thread_memberships, :original_message)
         .joins(
-          "LEFT JOIN chat_messages AS last_message ON last_message.id = chat_threads.last_message_id",
+          "LEFT JOIN user_chat_thread_memberships ON chat_threads.id = user_chat_thread_memberships.thread_id AND user_chat_thread_memberships.user_id = #{guardian.user.id} AND user_chat_thread_memberships.notification_level IN (#{::Chat::UserChatThreadMembership.notification_levels[:muted]})",
```

Comment:
The idea here is that the membership is optional and we should still list the thread if there's not membership, but if there's a membership we want to have it to use it for ordering

Reply:
I could sanitize sql here, but user id and notifications levels are not supposed to be risky

Reply:
I think it's totally fine, these are not user-provided values :+1: